AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/config/CustomMessages.java

```diff
@@ -501,12 +607,60 @@ public static void failable(
         @NotNull final String path,
         @NotNull final String errorPath,
         @NotNull final BooleanSupplier isError,
-        final String... placeholders
+        final Object... placeholders
```

Comment:
We can probably use TagResolvers instead of objects in this case so we can resort to altered display names and such.